fe/fe-core/src/test/java/org/apache/doris/nereids/jobs/CostAndEnforcerJobTest.java

```diff
@@ -1,102 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
why remove it?

Reply:
It's redundant, other UT have test the whole progress.